7.1/Dockerfile

```diff
@@ -19,20 +19,22 @@ LABEL io.k8s.description="Nginx and PHP 7 (FPM)" \
 RUN curl 'https://setup.ius.io/' -o setup-ius.sh && \
     bash setup-ius.sh
 RUN rm setup-ius.sh
+RUN yum update -y && yum clean all -y
 RUN yum install -y --setopt=tsflags=nodocs --enablerepo=centosplus \
-    php70u-fpm-nginx \
-    php70u-cli \
-    php70u-gd \
-    php70u-imap \
-    php70u-json \
-    php70u-mbstring \
-    php70u-mcrypt \
-    php70u-opcache \
-    php70u-pdo \
-    php70u-pdo \
-    php70u-xml
-#RUN rpm -V nginx php70w-cli php70w-fpm php70w-opcache
-RUN yum clean all -y
+    php71u-fpm-nginx \
+    php71u-cli \
+    php71u-gd \
+    php71u-imap \
+    php71u-json \
+    php71u-mbstring \
+    php71u-mcrypt \
+    php71u-bcmath \
```

Comment:
added bcmath to speed up BigInteger math to be able to run thinbus php SRP crypo.